news/_posts/2019-07-04-1.6.1.x-what-s-next.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "1.6.1.x: what’s next"
+subtitle: "The end of 1.6 official support by the PrestaShop company. And the beginning of something new."
+date:   2019-07-04 11:30:00
+authors: [ AntoineThomas ]
+icon: icon-flag-corner
+tags:
+ - 1.6
+ - Maintenance
+---
+
+
+This last year, the PS company has been facing a dilemma: how to dedicate 100% of the developers’ time to move forward faster on the 1.7 version of the open source project, something everybody agrees on. And, at the same time, provide more time for merchants still using the 1.6 version to get ready to upgrade, by keeping them safe in case of critical issue [after the official support ends on the 30th of June 2019[(https://www.prestashop.com/en/blog/maintenance-extension-prestashop-1-6). We have come up with a solution, thanks to the initiative and support of a few volunteers from the PrestaShop developer community: allowing them to take over the responsibility of maintaining PrestaShop 1.6.1.x beyond the end of official support.
+
+
+## What does it mean?
+
+Put it simply, from the 1st of July 2019, the maintenance of PrestaShop 1.6.1.x is now being performed exclusively by volunteers. If a critical issue is discovered (we think mainly of security vulnerabilities), this team of volunteers will look for solutions from the community, work on it through a process of code review and extensive testing, in order to release a patch version to solve this critical issue.
```

Comment:
I really prefer to talk about the 1.6 and not the 1.6.1.x which can be confusing for the community.
Anyway, they will have to build a new release if they fix something